src/lib/openapi/spec/feature-tag-schema.ts

```diff
@@ -35,6 +35,12 @@ export const featureTagSchema = {
             description:
                 'The value of the tag. This property is deprecated and will be removed in a future version of Unleash. Superseded by the `tagValue` property.',
         },
+        createdByUserId: {
+            type: 'number',
+            nullable: true,
```

Comment:
Just a note so we don't forget: maybe we need to make this not nullable, but if this is used as input it might be a breaking change for the API

Reply:
Yeah this is part of import/export. So I think it needs to be nullable